addons/binding/org.openhab.binding.wink/src/main/java/org/openhab/binding/wink/client/WinkClient.java

```diff
@@ -31,6 +31,15 @@ public static synchronized IWinkClient getInstance() {
     }
 
     /**
+     * Return a new instance of a wink client
+     *
+     * @return
+     */
+    public static IWinkClient newInstance() {
```

Comment:
You don't seem to use this?

Reply:
ah...good point.  I was having some classloader problems with the ClientBuilder.newClient() method so I started playing around and then forgot to remove this.  I'll kill it.